bb8/src/api.rs

```diff
@@ -165,8 +165,11 @@ impl<M: ManageConnection> Builder<M> {
     ///
     /// Defaults to None.
     #[must_use]
-    pub fn min_idle(mut self, min_idle: Option<u32>) -> Self {
-        self.min_idle = min_idle;
+    pub fn min_idle<U>(mut self, min_idle: U) -> Self
```

Comment:
Nit: if we're going to do this, I'd like it spelled `min_idle: impl Into<Option<u32>>` (same for the others).

Reply:
Oh, I had never done it like that, but actually that looks pretty good.